.github/workflows/ci.yml

```
@@ -380,17 +380,58 @@ jobs:
         shell: pwsh
         working-directory: webapp
         run: npm run check
-      
-      - name: check recording player
-        shell: pwsh
-        working-directory: webapp
-        run: npm run player:check
 
       - uses: actions/upload-artifact@v4
         with:
           name: webapp-client
           path: webapp/client/
 
+
+  devolutions-gateway-player:
+    name: Recording Player
+    runs-on: ubuntu-latest
+    needs: preflight
+    if: github.event_name != 'pull_request' || github.event.pull_request.head.repo.full_name == github.repository
+
+    steps:
+      - name: Checkout ${{ github.repository }}
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ needs.preflight.outputs.ref }}
+
+      - name: Check out Devolutions/actions
+        uses: actions/checkout@v4
+        with:
+          repository: Devolutions/actions
+          ref: v1
+          token: ${{ secrets.DEVOLUTIONSBOT_TOKEN }}
+          path: ./.github/workflows
+
+      - name: Setup .npmrc config file
+        uses: ./.github/workflows/npmrc-setup
+        with:
+          npm_token: ${{ secrets.ARTIFACTORY_NPM_TOKEN }}
+
+      - name: Install Biome
+        run: npm install -g @biomejs/biome
+
+      - name: Check recording player
+        working-directory: webapp/player-project
+        run: biome check ./src
+
+      - name: Install recording player
```

suggestion: Confusing naming

```suggestion
      - name: Install NPM dependencies
```